### PR TITLE
[EdgeTPU] Add EdgeTPU backend registration codes

### PR DESCRIPTION
**Pull request by rhqjatn2398**
This commit adds EdgeTPUToolchain in extension.ts using API.registerBackend() and updates the testing codes.

ONE-vscode-DCO-1.0-Signed-off-by: Bumsoo Ko <rhqjatn2398@naver.com>

### CODE REVIEW

**Review by jyoungyun — APPROVED** [review]
LGTM

**Comment by profornnan** [comment]
LGTM :D

**Review by profornnan — APPROVED** [review]

**Review by dayo09 — APPROVED** [review]
👍

File: src/Tests/Toolchain/ToolchainProvider.test.ts [hunk 1]
```diff
@@ -366,4 +368,10 @@ suite("Toolchain", function () {
       });
     });
   });
+
+  teardown(function () {
```
Comment (dayo09) [inline src/Tests/Toolchain/ToolchainProvider.test.ts:372]: Thanks that remembering the teardown location issue and relocate them :-D!

For the other reviewers: https://github.com/Samsung/ONE-vscode/issues/1678

**Pull request merged**